docs/library/dolphin.md

```diff
@@ -126,29 +116,36 @@ Frontend-level settings or features that the Dolphin core respects.
 
 ## Directories
 
-The Dolphin core's library name is 'Dolphin'
-
-TODO/FIXME
+In addition to the aforementioned 'dolphin-emu' directory, the dolphin-libretro core also creates a folder structure similar to that used by the standalone Dolphin emulator in RETROARCH_SAVES_FOLDER/User. In this structure, you can access/edit most of the same functionality you would find in the standalone Dolphin emulator and can even copy some files--such as GAME.ini and save files--back and forth between the dolphin-libretro core and the standalone Dolphin emulator.
```

Comment:
> RETROARCH_SAVES_FOLDER/User

With "Sort Saves into Folders by Core Name" being enabled by default now in RetroArch, that would be RETROARCH_SAVES_FOLDER/dolphin-emu/User.

For the rest, LGTM! :+1: